python/python/res/job_queue/job_manager.py

```diff
@@ -336,6 +336,7 @@ def postMessage(self, job=None, extra_fields={}, url=None):
 
         payload = {"user": self.user,
                    "cwd": os.getcwd(),
+                   "application": "ert",
```

Comment:
It is certainly the same "system". The reason I wanted to differentiate is that the type of problems which are logged by the main ert process and the forward model jobs is *widely* different.

> When I go looking at my Ert-logs is it not natural that I see the logs for both the main ert process and the forward models?

That might be - but it is certainly important that I can easily differentiate between the two. As long as that differentiation is simple and possible I have no strong opinions.